config/locales/helpers.en.yml

```diff
@@ -61,6 +63,9 @@ en:
         add_another_options:
           true: "Yes"
           false: "No"
+    placeholder:
+      assessor_interface_assessment_section_form:
+        failure_reason_notes: "Example: The right-hand section of your teaching qualification document is missing, please take a new image and upload it."
```

Comment:
```suggestion
        failure_reason_notes: Example: The right-hand section of your teaching qualification document is missing, please take a new image and upload it.
```

Reply:
Do i not need the speech marks here because of 'example:'?